File: C++/Iniciante/1009.cpp
#include <stdio.h>
#include <iostream>

int main() {
  
  std::string nome;
  double salario, vendas;

  std::cin >> nome;
  std::cin >> salario;
  std::cin >> vendas;

  salario += vendas * 0.15;

  printf("TOTAL = R$ %.2lf\n", salario);
  
}

File: Elixir/Iniciante/1009.exs
nome = IO.gets("") |> String.trim()
salario = IO.gets("") |> String.trim() |> String.to_float
vendas = IO.gets("") |> String.trim() |> String.to_float

salario = salario + vendas * 0.15

IO.puts "TOTAL = R$ #{Float.to_string(salario, decimals: 2)}"
